docs/_docs/installation/macos.md

```diff
@@ -3,156 +3,84 @@ title: Jekyll on macOS
 
-# Install Ruby
-brew install ruby
-```
+The instructions below are an excerpt from this detailed external guide to [install Ruby on Mac](https://www.moncefbelyamani.com/how-to-install-xcode-homebrew-git-rvm-ruby-on-mac/). They work best if you're setting up development tools for the first time on your Mac. If you've already tried to install Ruby or Jekyll on your Mac, or if you run into any issues, read that guide. 
 
-Add the brew ruby and gems path to your shell configuration:
+### Step 1: Install Homebrew
 
-```bash
-# If you're using Zsh
-echo 'export PATH="/usr/local/opt/ruby/bin:/usr/local/lib/ruby/gems/3.0.0/bin:$PATH"' >> ~/.zshrc
-
-# If you're using Bash
-echo 'export PATH="/usr/local/opt/ruby/bin:/usr/local/lib/ruby/gems/3.0.0/bin:$PATH"' >> ~/.bash_profile
-
-# Unsure which shell you are using? Type
-echo $SHELL
-```
-
-Relaunch your terminal and check your Ruby setup:
+[Homebrew](https://brew.sh/) makes it easy to install development tools on a Mac.
 
 ```sh
-which ruby
-# /usr/local/opt/ruby/bin/ruby
-
-ruby -v
-{{ site.data.ruby.current_version_output }}
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
 ```
 
-You're now running the current stable version of Ruby!
+### Step 2: Install chruby and the latest Ruby with ruby-install
 
-### With rbenv {#rbenv}
-
-People often use [rbenv](https://github.com/rbenv/rbenv) to manage multiple
-Ruby versions. This is very useful when you need to be able to run a given Ruby version on a project.
+Install `chruby` and `ruby-install` with Homebrew:
 
 ```sh
-# Install Homebrew
-/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
-
-# Install rbenv and ruby-build
-brew install rbenv
-
-# Set up rbenv integration with your shell
-rbenv init
-
-# Check your installation
-curl -fsSL https://github.com/rbenv/rbenv-installer/raw/main/bin/rbenv-doctor | bash
+brew install chruby ruby-install
 ```
 
-Restart your terminal to apply your changes.
-Next, you can install the Ruby version you want. Let's install the latest stable version:
+Install the latest stable version of Ruby:
 
 ```sh
-rbenv install {{ site.data.ruby.current_version }}
-rbenv global {{ site.data.ruby.current_version }}
-ruby -v
-{{ site.data.ruby.current_version_output }}
+ruby-install ruby
 ```
 
-That's it! Head over to [rbenv command references](https://github.com/rbenv/rbenv#command-reference) to learn how to use different versions of Ruby in your projects.
-
-## Install Jekyll
-
-After installing Ruby, install Jekyll and Bundler.
-
-### Local Install
-
-Install the bundler and jekyll gems:
+This will take a few minutes, and once it's done, configure your shell to automatically use `chruby`:
 
 ```sh
-gem install --user-install bundler jekyll
+echo "source /usr/local/opt/chruby/share/chruby/chruby.sh" >> ~/.zshrc
+echo "source /usr/local/opt/chruby/share/chruby/auto.sh" >> ~/.zshrc
```

Comment:
What do you think about using `brew --prefix` here instead?
```suggestion
echo "source $(brew --prefix)/opt/chruby/share/chruby/chruby.sh" >> ~/.zshrc
echo "source $(brew --prefix)/opt/chruby/share/chruby/auto.sh" >> ~/.zshrc
```

Reply:
Yes! I had the same thought but forgot to make the change. I'll also remove the next line.